### DISCUSSION TITLE
zigbee router connect to another router it's possible ?

### DISCUSSION THREAD

**Discussion by devildant** [Koenkk/zigbee2mqtt#19251] [General]
Hi,

I got into home automation for a few weeks now, and my list of Zigbee devices is increasing drastically, I use a Sonoff key but from what I have seen it is limited to 32 direct devices, a problem for many of my device are routers, and it connects directly to the coordinator, even if I specified a specific router for the association, it always connects to the coordinator.
Is it possible to connect the routers together without making a direct connection with the coordinator?

**Comment by francisp2** [comment]
Yes, but you can't force Zigbee 3 devices to use a specific route, the will find the best route themselves.

**Reply by devildant** [reply]
Thx for your reply, it's a shame, this is nevertheless part of a fairly basic logic, likewise it does not seem possible to transform a router into an end device.

so there is no workaround when we reach the maximum of direct device, unless we create another instance of z2m?

**Reply by francisp2** [reply]
I'm sure when the coordinator can't accept any more children, new devices will route through your routers nicely.

**Reply by devildant** [reply]
that would be good news :) but I'm still perplexed 😛
